sulong/tests/com.oracle.truffle.llvm.tests/src/com/oracle/truffle/llvm/tests/interop/MembersTest.java

```diff
@@ -41,19 +41,26 @@
 public class MembersTest extends InteropTestBase {
```

Comment:
I'd create a new test class for testing C++ members (e.g. `CxxMembersTest`). Not a blocker, though.